### DISCUSSION TITLE
How to remove a default existing which-key binding?

### DISCUSSION THREAD

**Discussion by mpas** [LazyVim/LazyVim#1186] [Q&A] [answered]
I want to cleanup my `which-key` keys a bit and tried several combinations to unbind a key defined in the default but I am not able to do this. 

I tried to remove the "Changelog" keybind in the following way..

```
  {
    "folke/which-key.nvim",
    opts = function()
      require("which-key").register({
        ["<leader>L"] = nil,
      })
    end,
  },

```

Also tried to set it to an empty string.

Any hint on how to remove an existing `which-key` bind would be appreciated!

**Comment by jyuan0** [comment, marked as answer]
Try adding `vim.keymap.del("n", "<leader>L")` to `config/keymaps.lua`.

**Reply by mpas** [reply]
Thanks a lot!! It works!
